src/Security/Login.cpp

```diff
@@ -91,6 +91,7 @@ std::string Login(const std::string& fields) {
         if (d.contains("message")) {
             d.erase("private_key");
             d.erase("public_key");
+            info(d["message"]);
```

Comment:
What about `User login result: `? This code also runs when the login succeeded.

Reply:
Print it only when the login fails, otherwise it could lead to confusion.

Reply:
> Print it only when the login fails, otherwise it could lead to confusion.
If the login is successful there's a message with the username. In my opinion that can be useful should be printed.

Regarding your second comment, What is the point of erasing the keys if the json document goes out of scope anyway when `message` doesn't exist?